demos/mouserotation/rotation.go

```diff
@@ -0,0 +1,46 @@
+package main
+
+import (
+	"image/color"
+
+	"engo.io/ecs"
+	"engo.io/engo"
+	"engo.io/engo/demos/demoutils"
+)
+
+type DefaultScene struct{}
+
+var (
+	// rotationSpeed is the speed at which to rotate
+	rotationSpeed float32 = 10
+
+	worldWidth  int = 800
+	worldHeight int = 800
+)
+
+func (*DefaultScene) Preload() {}
+
+// Setup is called before the main loop is started
+func (*DefaultScene) Setup(w *ecs.World) {
```

Comment:
I'm on a laptop, I don't have a middle mouse button, I'll try it out with my mouse later.